static/app/components/replaysOnboarding/utils.tsx

```diff
@@ -2,6 +2,11 @@ import {replayFrontendPlatforms} from 'sentry/data/platformCategories';
 import platforms from 'sentry/data/platforms';
 import type {PlatformIntegration} from 'sentry/types/project';
 
-export const replayJsFrameworkOptions: PlatformIntegration[] = platforms.filter(p =>
-  replayFrontendPlatforms.includes(p.id)
-);
+export function replayJsFrameworkOptions(): PlatformIntegration[] {
+  const frameworks = platforms.filter(p => replayFrontendPlatforms.includes(p.id));
+  const jsPlatformIdx = frameworks.findIndex(p => p.id === 'javascript');
+  const jsPlatform = frameworks[jsPlatformIdx];
+  frameworks.splice(jsPlatformIdx, 1);
+  frameworks.unshift(jsPlatform);
+  return frameworks;
```

Comment:
had to do this logic bc `platforms` (used across the codebase so don't want to mess with it) is sorted alphabetically :(